### ISSUE TITLE
Different feature values in caffemodel and mxnet model

### ISSUE THREAD

**Issue by onzone** [apache/incubator-mxnet#6269]
## Environment info
Operating System: Windows

Compiler: Microsoft Visual Studio VC14

Package used (Python/R/Scala/Julia): Python,C++

MXNet version: 0.9.5

Python version and distribution: 2.7

First of all I am a newbie in the mxnet. There have been two issues when I tried to incorporate mxnet in my project. I used to work in caffe before.
1. So for testing, I downloaded pretrained caffemodel and deploy.prototxt of Resnet-50 from [https://onedrive.live.com/?authkey=%21AAFW2-FVoxeVRck&id=4006CBB8476FF777%2117887&cid=4006CBB8476FF777] and converted the model into mxnet ndarray using mxnet/tools/caffe_converter/convert_model.py. Then I have extracted  feature of 2048 dimension from pool5 layer in both caffe and mxnet using both the python and C++. But the values of the feature vector are way too different. I am  providing the python and c++ code which I have used to extract the feature. May I know why is there so much difference in values in the feature vector? 
I have tried with pretrained vgg-face model too, but there also I keep getting different values in caffe and mxnet.

2. Also a strange thing noticed when I wanted to convert a resnet-50 caffemodel that I have trained in caffe with my own datasets. The caffemodel works perfectly well in extracting features in caffe. But when i tried to convert it into mxnet, there were no issues in conversion, but I keep getting same feature vector for every image(even I checked with blank image). When I digged into the network outputs, I saw after two-three layer of convolution and batch-normalization layers the outputs become similar for any input. Can you point me into any direction where I should look into to fix this error.

[feature_extract_mxnet_cpp.txt](https://github.com/dmlc/mxnet/files/1003892/feature_extract_mxnet_cpp.txt)
[feature_extract_mxnet_python.txt](https://github.com/dmlc/mxnet/files/1003893/feature_extract_mxnet_python.txt)

**Comment by yajiedesign** [comment]
This issue is closed due to lack of activity in the last 90 days. Feel free to reopen if this is still an active issue. Thanks!
